### DISCUSSION TITLE
Potential Actionable Items after RSEAA2023

### DISCUSSION THREAD

**Discussion by rowlandm** [RSEAA/RSEAA.github.io#65] [Discussion topics for #RSEAA2023]
This is to think about what we can do after the unconference. Please add in any ideas that have come up in the previous discussion topics and add them in the comments with the word ACTION: so that they are easily seen and everyone can vote for it. 

You can also add in comments underneath the ACTION comments if you have anything extra to contribute.

We may share this with people outside the unconference to get more input.

===================== From 2022 =====================
To start off, the initial presentation suggested two key actions:
holding this event again in 2023 and starting a community
discussion to make the RSE-AUNZ association a legal entity.

In room 1 (Community Building), we had 15 participants that
discussed the ideas such as piggy-backing off existing events
such as training to introduce community elements, hold broadbased events which are likely to achieve critical mass within the
institution e.g. get your (RSE) development questions answered,
use focussed events e.g. in Bioinformatics (or even particular
Bioinformatics systems) that will bring in the relevant RSEs.

In room 2 (Peer programming and collaboration) we had 9
participants that talked about an idea of peer programming
across the RSE community and the idea of a multi-institution
hacky hour where researchers could ask questions to a national
audience of RSEs to get help.

In room 3 (Unconference 2023 ideas) we did not have enough
participants and so the room was closed.

While it was not brought up in this meeting, in session two the
idea of local chapters catching up with each other could be
added to the ideas provided in room 1.

**Comment by rowlandm** [comment]
ACTION: Should we do this unconference next year?

**Reply by supernord** [reply]
Yes!

**Reply by richarddushime** [reply]
Yes

**Reply by manodeep** [reply]
Yes

**Reply by chesadi** [reply]
Yes

**Reply by robmoss** [reply]
Yes! And please keep the half-day format, it's extremely helpful

**Comment by rowlandm** [comment]
ACTION: Should we do a RSE team leader event / workshop / training? eg. How do RSE groups engage with researchers and prioritise tasks and Service Level Agreements (SLAs)?

**Reply by manodeep** [reply]
@likeajumprope made an excellent observation that there may be different perspectives between (formal) team leaders vs (informal) community leaders.

**Comment by pr4deepr** [comment]
ACTION: Collate and have a central list of training resources for RSEs

**Reply by supernord** [reply]
https://software-carpentry.org/

**Comment by supernord** [comment]
ACTION: Identify practical next steps to improve visibility of software created by RSEs

**Reply by supernord** [reply]
CITATION.cff usage, for example
- Website: https://citation-file-format.github.io
- Wizard: https://citation-file-format.github.io/cff-initializer-javascript/#/

**Reply by supernord** [reply]
FAIR evaluation tools like the FAIRsoft Evaluator from OpenEBench
https://openebench.bsc.es/observatory/

**Comment by pr4deepr** [comment]
ACTION: Identifying ways how to communicate and share computational paradigms and resources across domains in a domain agnostic manner.

https://github.com/RSEAA/RSEAA.github.io/discussions/56

**Comment by robmoss** [comment]
ACTION: Put together an overview of the SRE contributions and outputs (software, papers, grants, etc) across our RSEAA community

I feel this is relevant to a few different topics that came up for discussion, from visibility of RSE activities, to finding similar problems/methods across different research disciplines, and demonstrating what this community is capable of.

**Reply by pr4deepr** [reply]
This is a great idea.. will help communicate ideas across domain..
Could we use a resource  to link problems across domain? (knowledge graph maybe? not sure of the terminology)

**Reply by supernord** [reply]
A platform like a workflow registry could also work
Or a Zenodo community for more general applications?

**Reply by robmoss** [reply]
I'm not sure exactly what this would look like, to be honest. And the various discussion topics I mentioned might need different formats or outputs.

**Comment by SaranjeetKaur** [comment]
ACTION: There is a RSE Worldwide community handbook coming up (I was in the planning team for it at RSECon23, UK) - perhaps we can contribute to its documentation next year onwards.

**Comment by manodeep** [comment]
ACTION: Synthesise documents of existing RSE careers from Curtin/CSIRO/MDAP for different levels and associated pay-scaling, that can then be used as a starting point by other institutions

**Comment by manodeep** [comment]
ACTION: Create a tool (aimed at junior researchers *NOT* RSEs) that ingests python scripts and outputs an (incomplete) list of dependencies and then uses citeas.org to generate the (incomplete) list of citations

**Reply by manodeep** [reply]
This links back to the action proposed by @supernord about improving the [visibility of software](https://github.com/RSEAA/RSEAA.github.io/discussions/65#discussioncomment-7009980)

**Reply by tom-h** [reply]
I hope you don't mind @manodeep , but I'll take this idea to the CZI hackathon in a few weeks time as a suggestion for something to develop

**Reply by manodeep** [reply]
@tom-h Sure thing! But please do include in the discussion/development of such a citation tool (`citer`?)

**Comment by rowlandm** [comment]
ACTION: Is there a way we can reward RSEs via the RSE community? So for example, could we do a "RSE review" of someone's project and commend them for doing XYZ really well? eg. This project is a great example of glue code and is considered an exemplar of the RSE community because of ABC.

**Reply by pr4deepr** [reply]
I think "featuring" the work would be a good way. 
Also, if there is potential to offer small travel grants or awards that would be good way to feature and appreciate the work. Doesn't need to be large amounts

**Reply by rowlandm** [reply]
Look at certified professional for ACS and IEEE.

**Reply by nrmay** [reply]
@rowlandm @pr4deepr 
We could set up a version of the _Research Software Encyclopedia_ for Australasia. This can be deployed on a git repository and allows RSE to add their entries for their software and to categories theirs, and other people's, software. The encyclopedia could be expanded to include a feature that supports voting in some format. Then the _top_ projects could be shared and highlighted around the community.

See [the article _The Research Software Encyclopedia_](https://www.researchgate.net/publication/359015153_The_Research_Software_Encyclopedia_A_Community_Framework_to_Define_Research_Software) and an example of [a research catalogue](https://rseng.github.io/software/).

This would improve the visibility of RSE's work and provide an opportunity for the community to help define _research software_.

**Reply by supernord** [reply]
Some more examples 😄 
- ToolFinder: https://australianbiocommons.github.io/2_tools.html
- bio.tools: https://bio.tools/
- Research Software Directory: https://research-software-directory.org/software

**Reply by tom-h** [reply]
At the ARDC, we feel we are doing this through our "[Shaping Research Software](https://ardc.edu.au/news-and-events/news/?_keyword=%22Shaping%20Research%20Software%22&_categories=research-software)" series, and through our [sponsorship of awards](https://ardc.edu.au/get-involved/ardc-sponsored-awards-and-prizes/). But we need help from the RSE community to locate the right people to celebrate, and also for those areas that might benefit from a sponsored award.

**Comment by rowlandm** [comment]
ACTION: Hacky Hour meeting (maybe once a quarter) across RSEs so people could ask questions about RSE stuff

**Reply by supernord** [reply]
BioCommons communities meet quarterly, and we try to have presentations from community members
e.g. https://www.biocommons.org.au/events/workflows-community-meeting

**Comment by rowlandm** [comment]
ACTION: Get the Total Cost of Research to include RSEs. Be proactive to get into discussions before the grant.

**Reply by MelanieJH** [reply]
I strongly endorse this comment. What we do at the Curtin Institute for Data Science is exactly this, where possible we work with researchers to help advise, scope, and appropriately cost any research software contributions for a project before a grant is submitted. We have also developed a good relationship with our Research Office so that when they get grants come in for review, if they notice there is a software component which hasn't been discussed with us they will suggest researchers might want to consult before submission. Note it's only a suggestion, not a requirement as it is important for researchers to have other avenues to scope or cost software components. Developing an awareness of the importance and total cost of software components in institutional research offices is just as or probably more important than doing it with researchers.

**Comment by pdart** [comment]
Action: Choose some software libraries/platforms developed locally, help make them as citeable (acknowledgeable?) as possible, try to track usage through to results (may want to use examples that track users in some way), and ensure that appropriate credit is given to the software/RSEs. We could partner with other RSE orgs to do this. We can use this generate guidance to RSEs re impact measures/getting recognition, and to provide a basis for lobbying journals, funding agencies and other organisations about appropriate recognition.

**Reply by supernord** [reply]
This is similar to what we have tried with workflows on WorkflowHub
Registering best practice there and then trying to track when the workflow DOIs appear in the literature.

**Reply by supernord** [reply]
e.g. https://workflowhub.eu/programmes/8/workflows

**Reply by robmoss** [reply]
@supernord am I right in thinking a different platform would be needed for software packages that aren't workflows?

**Reply by supernord** [reply]
Zenodo, as it integrates with github?

**Reply by supernord** [reply]
And auto-generates DOIs for software releases

**Reply by robmoss** [reply]
Fair. I'm frustrated that zenodo doesn't support platforms like GitLab (and institutional instances)

**Reply by robmoss** [reply]
Oh, I was also thinking of a platform that provides more of a gallery or showcase than zenodo does (or appears to do, given my limited experience with it). WorkflowHub looks like it has more structure and discoverability.

**Reply by supernord** [reply]
you can create communities on Zenodo

**Reply by robmoss** [reply]
I did not know that, thank you!

**Reply by supernord** [reply]
It might be good simply to share known approaches for different types of software
And why they are useful

**Reply by tom-h** [reply]
@robmoss while Zenodo doesn't integrate with those others, it is still possible to just upload a tarball. But, that's no fun for software with releases. To my knowledge, UTS was the only uni that tried to set up a connection between their institutional gitlab instance and their institutional data repo.

Also, re platforms, I'd be curious to get your reaction to RSD ([Research Software Directory](https://research-software-directory.org/)). It is supposed to be used to showcase research software projects, and so has hooks to things like literature.

**Reply by robmoss** [reply]
@tom-h yeah, I'm not excited at the prospect of uploading tarballs. Darn!

Thanks very much for the RSD link! I'm not sure whether I previously knew about this — if so, it was a long time ago and it's clearly slipped my mind. This looks like a really nice platform, and it solves/addresses [my suggested action](https://github.com/RSEAA/RSEAA.github.io/discussions/65#discussioncomment-7009995). I'll have to email them, arrange an account, and list some of my packages.

**Comment by MelanieJH** [comment]
As I'm not part of the group I won't list this as an action, but I think it would be very valuable for this group to follow up on getting some senior people to help in an advisory capacity as discussed on the strategy panel yesterday and in your strategic Goal 4 to map out a mature organisational structure. If you do choose to do this, I'd be willing to contribute.

**Comment by rowlandm** [comment]
ACTION: Discussion of what would be needed for applying for a Centre of Excellence for Scientific Software

**Reply by manodeep** [reply]
Might be worthwhile to get feedback from the folks that proposed the Data Science CoE that got knocked back by the ARC ~8 years ago

**Reply by jni** [reply]
Got any links about that @manodeep? First I hear of it. 😃

**Reply by manodeep** [reply]
@jni Not sure that there is anything publicly available, given that the proposal got rejected (not even sure at what round, I was obviously not involved). I will ask around to see if I can gather any further info

**Comment by MelanieJH** [comment]
Have a look at the assessment criteria for CoEs, they are focused to demonstrated research excellence via standard metrics like publications and need senior researchers to lead. You might consider other ARC programs like an ITTC which is focused on addressing skills gaps and is potentially a better fit for research software.

-Melanie

Sent from my iPhone

On 28/09/2023, at 11:34 AM, Manodeep Sinha ***@***.***> wrote:

﻿

Might be worthwhile to get feedback from the folks that proposed the Data Science CoE that got knocked back by the ARC ~8 years ago

**Reply by MelanieJH** [reply]
Discussion of assessment criteria are here: https://www.arc.gov.au/funding-research/funding-outcome/selection-outcome-reports/Selection-Report%3A-ARC-Centres-of-Excellence-for-Funding-Commencing-in-2023

Please also be aware that a CoE bid requires a director with an outstanding track record of research in the area, as well as ideally managing large-scale multi-institutional teams and budgets. While not an official requirement, anecdotally when CoE bids are put together they seek a director who is already a Distinguished Professor at the lead institution and ideally a Fellow of one of the major academies, and/or previous ARC Laureate Fellow - if you look at past funded CoEs you can do your own correlation analysis and you can see the weightings in the criteria. These bids also require high levels of university co-investment so buy-in from DVC-Rs across a number of universities is essential.
